pkg/audit/service.go

```diff
@@ -73,9 +73,18 @@ func (service *Service) Close() error {
 
 // process picks a random stripe and verifies correctness
 func (service *Service) process(ctx context.Context) error {
-	stripe, err := service.Cursor.NextStripe(ctx)
-	if err != nil {
-		return err
+	var stripe *Stripe
+	for {
+		stripe, more, err := service.Cursor.NextStripe(ctx)
+		if err != nil {
```

Comment:
we could try to distinguish between different types of errors, but I do not think it would be worthwhile. Especially since it might be a list of varying errors

Reply:
oh sorry I missed to add the back ticks to the `more` word, hence my comment wasn't clear.

It wasn't about to distinguish the errors, it was related with the `more` returned value.

Reply:
But after reading the comment doc of `NewStripe` the question doesn't make sense.